### ISSUE TITLE
Non English words in tag name

### ISSUE THREAD

**Issue by Yachtsman** [DemocracyOS/app#641]
Hi there,

How can I create the tag in Russian? I'm receiving the following error for now.

```
{"error":{"message":"Validation failed","stack":"ValidationError: Path `hash` is required.\n at model.Document.invalidate (/home/kuchaeviv/Sevastopol.Social/app/node_modules/mongoose/lib/document.js:1021:32)\n at /home/kuchaeviv/Sevastopol.Social/app/node_modules/mongoose/lib/document.js:970:16\n at validate (/home/kuchaeviv/Sevastopol.Social/app/node_modules/mongoose/lib/schematype.js:610:7)\n at /home/kuchaeviv/Sevastopol.Social/app/node_modules/mongoose/lib/schematype.js:627:9\n at Array.forEach (native)\n at SchemaString.SchemaType.doValidate (/home/kuchaeviv/Sevastopol.Social/app/node_modules/mongoose/lib/schematype.js:614:19)\n at /home/kuchaeviv/Sevastopol.Social/app/node_modules/mongoose/lib/document.js:968:9\n at process._tickCallback (node.js:442:13)","name":"ValidationError","errors":{"hash":{"message":"Path `hash` is required.","name":"ValidatorError","path":"hash","type":"required","value":""}}}}
```

**Comment by jfresco** [comment]
Hi @Yachtsman. Please send questions to [our mailing list](http://groups.google.com/group/democracyos-app); this place is only for bug reports and feature requests.

The error you are posting is a validation error being thrown by the database engine and does not depends on our translation component. Perhaps in this case we should improve the error handling.
If you consider this a bug please open a new issue and describe what behavior you think is erroneous.
Closing now.

**Comment by gvilarino** [comment]
@Yachtsman we just released [version `0.10.10`](https://github.com/DemocracyOS/app/releases/tag/0.10.10) that support tag-names in non-`ASCII` format. Please update.

This may also be of interest to @tarasdudar
